conference.js

```diff
@@ -1827,5 +1773,37 @@ export default {
         if(room) {
             room.sendApplicationLog(JSON.stringify({name, value}));
         }
+    },
+    /**
+     * Disconnect from the conference and optionally request user feedback.
+     * @param {boolean} [requestFeedback=false] if user feedback should be
+     * requested
+     */
+    hangup (requestFeedback = false) {
+        const errCallback = (err) => {
+            // If we want to break out the chain in our error handler, it needs
+            // to return a rejected promise. In the case of feedback request
+            // in progress it's important to not redirect to the welcome page
+            // (see below maybeRedirectToWelcomePage call).
+            if (err === UIErrors.FEEDBACK_REQUEST_IN_PROGRESS) {
+                return Promise.reject('Feedback request in progress.');
+            }
+            else {
+                console.error('Error occurred during hanging up: ', err);
+                return Promise.resolve();
```

Comment:
Is anyone using these `Promise`s that are being returned? I skimmed to the differences for occurrences of `hangup` and didn't see anyone caring about the return value. If someone is using it, why is there no jsdoc about the return value?